src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_RER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCONT = 20;
+  const GENERAL_PRICE = days * PRICE_RER_DAY;
```

Comment:
even though this is declared with const, it's a calculated value, not a hardcoded constant
it should be separeted from constants and written id camelCase